client/layout/masterbar/item.tsx

```diff
@@ -221,49 +216,30 @@ class MasterbarItem extends Component< MasterbarItemProps > {
 			disabled: this.props.disabled,
 		};
 
-		if ( this.props.url && ! this.props.subItems ) {
```

Comment:
Moved from

```
if ( this.props.url && ! this.props.subItems ) {
  // render anchor tag
}

if ( this.props.url && this.props.subItems ) {
  // render button with anchor tag and sub-menu items as children
}

// else, if props.url is not defined
// render a wrapper div > button > menu items as children (no anchor tag)
```

to:
- always rendering a wrapper div
- render either a `button` or an `a` depending on props.url
- render the sub-menu items (when defined) as sibling of the `button`/`a`